models/deepseek_v4_flash_mtp/prefill_cp_csa.py

```
@@ -0,0 +1,3182 @@
+    }
+
+
+def build_tensor_specs(cp_size: int = CP_SIZE):
+    """Build the canonical CP-CSA fixture."""
+    from golden import TensorSpec
+
+    if cp_size != CP_SIZE:
+        raise ValueError(
+            f"runtime cp_size={cp_size} does not match static CP_SIZE={CP_SIZE}"
+    )
+    metadata, ctx = _build_metadata_tensors(cp_size)
```

Ruff (0.16.1)

[warning] Unpacked variable `ctx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)